.azure-pipelines/linux-conda-CI.yml

```diff
@@ -28,13 +28,21 @@ jobs:
         TENSORFLOW_PATH: tensorflow==1.15.0
         INSTALL_ORT: pip install onnxruntime==1.1.1
 
-      Python37-tf200:
```

Comment:
Remove this?

Reply:
Sure, since tf 2.0.0 is not in python 3.8, so we can remove this one.

Reply:
sure